R/dockerfile.R

```diff
@@ -27,38 +19,130 @@
 #' dockerfile()
 #'
 #' @import futile.logger
-dockerfile <- function(from = sessionInfo(), to = paste0(getwd(), "/", "Dockerfile")) {
-  flog.debug("Creating a new Dockerfile from %s to %s", from, to)
+dockerfile <- function(from = utils::sessionInfo(),env = NULL, maintainer = NULL, r_version = paste(R.Version()$major, R.Version()$minor, sep="."), image = NULL, context = NA_character_) {
```

Comment:
The user must be able to define `ENV` instructions somehow. Let's discuss that in person tomorrow.

I would much prefer to not "hide" the defaults in functions that fall back to something that is NULL. Put the defaults where they belong, into the function definition.